src/transformers/image_utils.py

```diff
@@ -377,7 +474,7 @@ def flip_channel_order(self, image):
 
         return image[::-1, :, :]
 
-    def rotate(self, image, angle, resample=PIL.Image.NEAREST, expand=0, center=None, translate=None, fillcolor=None):
+    def rotate(self, image, angle, resample=None, expand=0, center=None, translate=None, fillcolor=None):
```

Comment:
Moved to be within the function, rather than in the signature to avoid issues when importing and PIL isn't installed